image_creator/os_type/linux.py

```diff
@@ -129,6 +159,40 @@ def _remove_user_accounts(self):
             if self.image.g.is_dir(home) and home.startswith('/home/'):
                 self.image.g.rm_rf(home)
 
+    @sysprep('Renaming default cloud-init user to "%(default_user)s"')
```

Comment:
I'd suggest to disable this sysprep task by default (`enabled=False`) as it would be more intuitive for the users that are already familiar with the official cloud-init images of the distributions (e.g. in the Debian image the user might expect the default user to be "debian").